integrationTests/cypress/integration/cas_browser.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Browser-based CAS login and logout functionality
+
+  Scenario: logged out user is redirected to CAS login page
+    Given the admin is logged out of the CES
+    When the user opens the dogu start page
```

Comment:
the function which corrusponds to the string "the user opens the dogu start page" is visiting the host/redmine url. Therefore I would change it to  "the user opens the redmine dogu start page" because the function cannot be reused in other contexts, but its name suggests that it can be.

Reply:
Done